README.md

```diff
@@ -52,3 +52,25 @@ On top of the `atoti` package, it comes with:
   ```bash
   poetry run pylint app/ tests/
   ```
+
+### Deploying on Heroku
+
+The application can be deployed very easily on Heroku. Since we use poetry, you will need to add the following build packs using the CLI:
```

Comment:
Users should be the judge of whether this is _very_ easy or not ;).

```suggestion
The application can be deployed easily on Heroku.

Since the dependencies in this project are managed with Poetry, the following CLI commands are required to change the buildpacks:
```